mmaction/datasets/pipelines/formating.py

```diff
@@ -371,3 +372,31 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += f"(input_format='{self.input_format}')"
         return repr_str
+
+
+@PIPELINES.register_module()
+class FormatNtuPose:
+
+    def __init__(self, input_format):
+        self.input_format = input_format
+
+    def __call__(self, results):
+        keypoint = results['keypoint']
+        keypoint_confidence = results['keypoint_score']
+        keypoint_confidence = np.expand_dims(keypoint_confidence, -1)
+        keypoint_3d = np.concatenate((keypoint, keypoint_confidence), axis=-1)
+        keypoint_3d = np.transpose(keypoint_3d,
+                                   (3, 1, 2, 0))  # M T V C -> C T V M
+
+        if keypoint_3d.shape[-1] == 1:
```

Comment:
Add another argument named `num_person`, which indicates that you want to pad up to N persons.  (Instead of hardcoded 2 persons. )